assets/js/modules/ads/components/setup/SetupMainPAX.js

```diff
@@ -0,0 +1,234 @@
+					// We need to ensure data is resolved, otherwise the admin URL will be
+					// `undefined` and `finishSetup` will error instead of redirecting
+					// to the dashboard.
+					await registry
+						.__experimentalResolveSelect( CORE_SITE )
+						.getSiteInfo();
+
+					const adminURL = registry
+						.select( CORE_SITE )
+						.getAdminURL( 'googlesitekit-dashboard', {
+							notification: 'authentication_success',
+							slug: 'ads',
+						} );
+
+					finishSetup( adminURL );
+				}
+			}
+		},
+		[
+			setExtCustomerID,
+			setPaxConversionID,
+			submitChanges,
+			registry,
+			finishSetup,
+		]
+	);
+
+	const createAccount = useCallback( () => {
+		if ( ! hasAdwordsScope ) {
+			navigateTo( oAuthURL );
+		} else {
+			setShowPaxApp( 1 );
+		}
+	}, [ navigateTo, setShowPaxApp, hasAdwordsScope, oAuthURL ] );
+
+	return (
+		<div className="googlesitekit-setup-module googlesitekit-setup-module--ads">
+			<div className="googlesitekit-setup-module__step">
+				<div className="googlesitekit-setup-module__logo">
+					<AdsIcon width="33" height="33" />
+				</div>
+
+				<h2 className="googlesitekit-heading-3 googlesitekit-setup-module__title">
+					{ _x( 'Ads', 'Service name', 'google-site-kit' ) }
+				</h2>
+			</div>
+			<AdBlockerWarning />
+
+			{ ! isAdBlockerActive && !! showPaxApp && hasAdwordsScope && (
+				<PAXEmbeddedApp displayMode="setup" onLaunch={ onLaunch } />
+			) }
+
+			{ ! isAdBlockerActive && ( ! showPaxApp || ! hasAdwordsScope ) && (
+				<Fragment>
+					<div className="googlesitekit-setup-module__description">
+						{ createInterpolateElement(
+							__(
+								'Add your conversion ID below. Site Kit will place it on your site so you can track the performance of your Google Ads campaigns. <a>Learn more</a>',
+								'google-site-kit'
+							),
+							{
+								a: (
+									<SupportLink
+										path="/google-ads/thread/108976144/where-i-can-find-google-conversion-id-begins-with-aw"
+										external
+									/>
+								),
+							}
+						) }
+						<br />
+						{ __(
+							'You can always change this later in Site Kit Settings.',
+							'google-site-kit'
+						) }
+					</div>
+
+					<SetupForm
+						finishSetup={ finishSetup }
+						isNavigatingToOAuthURL={ isNavigatingToOAuthURL }
+						createAccountCTA={
+							<Fragment>
+								<SpinnerButton
+									onClick={ createAccount }
+									disabled={ isNavigatingToOAuthURL }
+									isSaving={ isNavigatingToOAuthURL }
+									inverse
+								>
+									{ __(
+										'Create an account',
+										'google-site-kit'
+									) }
+								</SpinnerButton>
+								{ ! hasAdwordsScope && (
+									<p className="googlesitekit-setup-module__permission-notice">
+										{ __(
+											'Additional permissions are needed to create a new Ads account. You’ll be asked to grant these permissions during the account creation process. Clicking the button will direct you to the authorization flow.',
```

Comment:
I updated the comment, since it is part of AC, we would need to confirm it with @aaemnnosttv regarding the suggestion to use a prompt, or not include the text. For me it makes more sense to have it as a text, so we can skip additional popup once user clicks on it, it is kinda easier for the user to set expectation early

Reply:
Fair enough, happy to leave the text for now, but let's update the text to my suggestion, just so it's shorter/easier to read 😄